### ISSUE TITLE
[med] Chrome + Jaws: New Group Page: Search result status is not being announced by screen reader.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8464]
## Action Performed:
1. Using Chrome + Jaws, open URL staging.new.expensify.com
2. Navigate to (+) button icon using Tab key and press Enter to select it.
3. Navigate and select new group using Enter & Tab key.
4. New group page is appearing, Input any name in the search editable text box.
5. Observe the screen reader announcement.

## Expected Result:
As soon as search result appear on the page, screen reader should announce as number of results found to the users.

## Actual Result:
After entering any input into the search field, some result will appear on the page but screen reader does not convey the result found information to the users.

**Other occurrences**
>Same issue repro on PR #8760

## Workaround:
Yes

## Area issue was found in:
New Group window

## Failed WCAG checkpoints
4.1.3

## User impact:
If any status message or any task completion information is not conveyed to the users then user will not able to identify whether task is completed or not and they require manual effort to identify the changes occur on the page.

## Suggested resolution:
Provide role=status to indicate the message like "number of result found" after performing the search operation which convey the status to the users.

Refer to:
1. https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html                                             
2. https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA22

## Platform:
- Web
- Android
- Mobile Web

**Version Number:**v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:** Stuti

https://user-images.githubusercontent.com/97600950/161549783-21453c5f-a38e-4367-af45-1295f96bc97e.mp4
